From 2eca8bd52a51fd74e0be362f5be6fd2c1951b872 Mon Sep 17 00:00:00 2001
From: Louis Marelle <louis.marelle@latmos.ipsl.fr>
Date: Wed, 8 Jun 2022 18:30:47 +0200
Subject: [PATCH 47/70] Fix radt synchronization in adaptive time step

In dyn_em/adapt_timestep_em.F, update the code synchronizing the radiative time
step to the adaptive time step, in order to remain consistent with changes in
b71e0a837626d3809ec6996f96035310329ea782
---
 dyn_em/adapt_timestep_em.F | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dyn_em/adapt_timestep_em.F b/dyn_em/adapt_timestep_em.F
index 1182fa9..b0147e0 100644
--- a/dyn_em/adapt_timestep_em.F
+++ b/dyn_em/adapt_timestep_em.F
@@ -402,7 +402,7 @@ RECURSIVE SUBROUTINE adapt_timestep(grid, config_flags)
              (radt_sec) ) * (radt_sec)
         CALL WRFU_TimeIntervalSet(tmpTimeInterval, S=rad_time)
         dtInterval = tmpTimeInterval - &
-             (domain_get_current_time(grid) - domain_get_sim_start_time(grid))
+             (domain_get_current_time(grid) - domain_get_start_time(grid))
 
         use_last2 = .TRUE.
         grid%stepping_to_time = .TRUE.
-- 
2.31.1

